src/common/fees/makeFees.ts

```diff
@@ -155,7 +162,7 @@ const fetchFeesFromVendor = async (
       processor: processEarnComFees
     })
     if (earnComFees != null) {
-      return earnComFees
+      return asFeeRatesCleaner(earnComFees)
```

Comment:
Notice that cleaners "throw" in case they fail. 
So you should probably either pass the cleaner to the "fetchFees" function and call it inside.